<launch>
    <node pkg="position_module" name="MT3339" type="SerialGPSDriver" output="screen">
        <param name="PortPath" value="/dev/ttyTHS2"/>
        <param name="ChipType" value="1" />
        <param name="frame_id" value="gps"/>
        <param name="BaudRate" value="9600"/>
        <param name="DataBit" value="8" />
        <param name="Parity" value="False" />
        <param name="StopBit_Even" value="False" />
        <param name="FlowCtrl" value="False" />
    </node>
</launch>